be/src/common/config.h

```diff
@@ -948,4 +948,11 @@ CONF_mInt64(load_tablet_timeout_seconds, "30");
 
 CONF_mBool(enable_pk_value_column_zonemap, "true");
 
+// timestamp unit to use for INT96-encoded timestamps in parquet.
+// SECOND, MICRO, MILLI, NANO
+// The default unit is MICRO, which is the precision of DATETIME/TIMESTGAMP in MySQL.
+// https://dev.mysql.com/doc/refman/8.0/en/datetime.html
+// A DATETIME or TIMESTAMP value can include a trailing fractional seconds part in up to microseconds (6 digits) precision
+CONF_mString(parquet_coerce_int96_timestamp_unit, "MICRO");
```

Comment:
Is it suitable as a config for BE? any situation we will make it different from each BE?

Reply:
The config item is moved to fe.conf.